src/main/kotlin/app/extractors/ExtractorInterface.kt

```diff
@@ -278,6 +278,10 @@ interface ExtractorInterface {
             return emptyList()
         }
 
+        if (languageLabel == "c" && languageLabel in selectedCategories) {
```

Comment:
What this block do?

Reply:
It is specific for C language.

Reply:
I see. But what it do? No lines libraries for c language?

Reply:
No, just for a line that has language label selected.

Reply:
Constant already defined: `if (languageLabel == CExtractor.LANGUAGE_NAME && ... )`
Also, add comments about percentile, please.